### PR TITLE
Feature/music album class

### PR DESCRIPTION
**Pull request by rtonata88**
In this PR I have done the following
- Created MusicAlbum class in a separate .rb file.
- Createx Genre class with an association to the Item class (in a separate .rb file).
- All MusicAlbum class properties visible in the diagram should be defined and set up in the constructor method.
- All Genre class properties visible in the diagram should be defined and set up in the constructor method.
- Implemented the following methods:
- `add_item` method in the Genre class which:
- `can_be_archived?()` in the MusicAlbum class

### CODE REVIEW

**Review by oliverSCZ — APPROVED** [review]
💯 Great Job 💯

File: Classes/music_album.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require './item'
+
+class MusicAlbum < Item
+  attr_reader :on_spotify
+
+  def initialize(publish_date, archived, on_spotify, id: nil)
+    super(publish_date, archived, id: id)
```
Comment (danalvrz) [inline Classes/music_album.rb:7]: Maybe would to use *args to add all inherited arguments from Item class.

File: Classes/genre.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :id, :name, :items
+
+  def initialize(id, name)
+    @id = id
```
Comment (danalvrz) [inline Classes/genre.rb:5]: Maybe don't add id as an argument for initialize

File: Classes/genre.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    items << item
```
Comment (danalvrz) [inline Classes/genre.rb:11]: I think you might be missing an @

File: Classes/item.rb [hunk 4]
```diff
@@ -21,3 +21,6 @@ def move_to_archive
     @archived = can_be_archived?
   end
 end
+
+item = Item.new(author: 'Richard Chambula')
+puts item.author
```
Comment (danalvrz) [inline Classes/item.rb:26]: To keep your code clean delete your test comments before pushing. 👍

**Review by danalvrz — CHANGES_REQUESTED** [review]
Great job, @rtonata88 ! There a minor changes, but the rest look great! Keep up the hard work 🥳

**Pull request merged**